src/App.css

```diff
@@ -1,42 +1,67 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+body {
+  background-color: aquamarine;
```

Comment:
Bold choice you might have more success with a theme from here https://www.happyhues.co/